From: Huang Rui <ray.huang@amd.com>
Subject: [PATCH v6 02/11] virtio-gpu: Configure new feature flag
 context_create_with_flags for virglrenderer
Date: Tue, 19 Dec 2023 15:53:11 +0800
Content-Type: text/plain

Configure a new feature flag (context_create_with_flags) for
virglrenderer.

Originally-by: Antonio Caggiano <antonio.caggiano@collabora.com>
Signed-off-by: Huang Rui <ray.huang@amd.com>
---

Changes in v6:
- Move macros configurations under virgl.found() and rename
  HAVE_VIRGL_CONTEXT_CREATE_WITH_FLAGS.

 meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/meson.build b/meson.build
index ec01f8b138..ea52ef1b9c 100644
--- a/meson.build
+++ b/meson.build
@@ -2287,6 +2287,10 @@
                        cc.has_member('struct virgl_renderer_resource_info_ext', 'd3d_tex2d',
                                      prefix: '#include <virglrenderer.h>',
                                      dependencies: virgl))
+  config_host_data.set('HAVE_VIRGL_CONTEXT_CREATE_WITH_FLAGS',
+                       cc.has_function('virgl_renderer_context_create_with_flags',
+                                     prefix: '#include <virglrenderer.h>',
+                                     dependencies: virgl))
 endif
 config_host_data.set('CONFIG_VIRTFS', have_virtfs)
 config_host_data.set('CONFIG_VTE', vte.found())
-- 
2.25.1



